docs/debops-playbooks/guides/getting-started.rst

```diff
@@ -193,9 +194,11 @@ To do that, in :file:`ansible/inventory/group_vars/all/sshd.yml` add:
    ---
    sshd__whitelist: [ '192.0.2.0/24', '2001:db8::/32' ]
 
-This will configure the [debops.ferm](https://docs.debops.org/en/latest/ansible/roles/ansible-ferm/docs/) and [debops.tcpwrappers](https://docs.debops.org/en/latest/ansible/roles/ansible-tcpwrappers/docs/) roles to allow connections to the :command:`ssh` service from specified networks.
+This will configure the :ref:`debops.ferm <debops.ferm>` and
+:ref:`debops.tcpwrappers <debops.tcpwrappers>` roles to allow connections to
+the :command:`ssh` service from specified networks.
 
-The [debops.sshd](https://docs.debops.org/en/latest/ansible/roles/ansible-sshd/docs/) role has `many more variables <https://docs.debops.org/en/latest/ansible/roles/ansible-sshd/docs/defaults.html>`_, you can checkout to see the default configuration used by DebOps and what can be changed as needed.
+The :ref:`debops.sshd <debops.sshd>` role has `many more variables <https://docs.debops.org/en/latest/ansible/roles/ansible-sshd/docs/defaults.html>`_, you can checkout to see the default configuration used by DebOps and what can be changed as needed.
```

Comment:
This should use an internal reference, but there's nothing good for now to point to. I'll add a reference later in the `debops.sshd` role to its defaults, and perhaps make this consistent in all roles if it works out.

For now, you could switch this and other URLs like this to point to `master` instead of `latest`, so that the branch and documentation build can be removed.